docs/manual/build_system/prerequisites.qbk

```diff
@@ -73,6 +73,11 @@ favorite compiler with __hpx__ visit __stellar_hpx_buildbot__.
       for compiling __hpx__ if you use gcc V4.6.x. Please use either
       an earlier or a later version of Boost with this compiler.]
 
+[important When compiling with the Intel Compiler on Linux systems, we only support
+      C++ Standard Libraries provided by gcc 4.6 and upwards. If the 'g++' in your
+      path is something below that requirement, please add this to your CMAKE_CXX_FLAGS
+      with the `-gxx-name=/path/to/g++` flag.]
+
```

Comment:
On 10/21/2015 01:20 PM, Hartmut Kaiser wrote:
> In docs/manual/build_system/prerequisites.qbk
> <https://github.com/STEllAR-GROUP/hpx/pull/1808#discussion_r42610928>:
> 
>> @@ -73,6 +73,11 @@ favorite compiler with __hpx__ visit __stellar_hpx_buildbot__.
>>        for compiling __hpx__ if you use gcc V4.6.x. Please use either
>>        an earlier or a later version of Boost with this compiler.]
>>  
>> +[important When compiling with the Intel Compiler on Linux systems, we only support
>> +      C++ Standard Libraries provided by gcc 4.6 and upwards. If the 'g++' in your
>> +      path is something below that requirement, please add this to your CMAKE_CXX_FLAGS
>> +      with the `-gxx-name=/path/to/g++` flag.]
>> +
> 
> What do you mean by 'add 'this' ? The correct version of g++? the library?

'this' was meant to refer to the CMAKE_CXX_FLAGS, do you have a better
formulation for this?

> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/STEllAR-GROUP/hpx/pull/1808/files#r42610928>.
> 


-- 
Thomas Heller
Friedrich-Alexander-Universität Erlangen-Nürnberg
Department Informatik - Lehrstuhl Rechnerarchitektur
Martensstr. 3
91058 Erlangen
Tel.: 09131/85-27018
Fax:  09131/85-27912
Email: thomas.heller@cs.fau.de